content/en/docs/reference/controllers/built-in.md

```diff
@@ -0,0 +1,155 @@
+---
+title: Built-in controllers
+content_template: templates/concept
+weight: 10
+---
+
+{{% capture overview %}}
+
+This page lists the {{< glossary_tooltip text="controllers" term_id="controller" >}}
```

Comment:
When I click on the controller tooltip it takes me to the kube-controller-manager page.   But this page really doesn't have a nice description of what controllers are and there basic function.   Where will you add content that provides an introductory overview of what controllers are and what their purpose is?

Reply:
I have a draft for an initial [overview page about Controllers](https://deploy-preview-15373--kubernetes-io-master-staging.netlify.com/docs/concepts/architecture/controller/) as part of PR #15373.

and I plan to propose that in a separate pull request. I'd also change the _controller_ glossary entry to point to that new page.

Reply:
The separate pull request about a controller concept is #15733

Reply:
Merged #15733